modules/flight_interface/flight_interface.py

```diff
@@ -15,9 +17,10 @@ class FlightInterface:
     __create_key = object()
 
     @classmethod
-    def create(cls, address: str) -> "tuple[bool, FlightInterface | None]":
+    def create(cls, address: str, timeout_home: float) -> "tuple[bool, FlightInterface | None]":
         """
         address: TCP or port.
```

Comment:
Is it worth specifying a type (typedef?) for TCP/PORT?

Reply:
No. Address is a path to a file (UNIX moment) (e.g. `"/dev/ttyUSB0"` ).